AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -913,7 +924,7 @@ - (void)rangeController:(ASRangeController *)rangeController didInsertSectionsAt
 - (void)rangeController:(ASRangeController *)rangeController didDeleteSectionsAtIndexSet:(NSIndexSet *)indexSet withAnimationOptions:(ASDataControllerAnimationOptions)animationOptions
 {
   ASDisplayNodeAssertMainThread();
-  
+  [_layoutFacilitator collectionViewEditingSectionsAtIndexSet:indexSet];
```

Comment:
@binl importantly, this has changed with @nguyenhuy's latest changes to reloading.  Although it is true that moves currently work that way, this is something that needs to be changed soon, as the current mechanism causes both incorrect / unpredictable animations and is less efficient than moving the actual node (and performing a relayout if it needs, but not actually destroying and recreating the nodes and thus forcing them to re-render).